core/io.c

```diff
@@ -1696,6 +1716,18 @@ int restart_init() {
 #if SCATTERING
       { init_all_hotcross(); }
 #endif
+
+      // Check boundary conditions and put photons where they belong
+      if (mpi_myrank() == 0) {
+        printf("Shuffling and load balancing particles\n");
```

Comment:
Is this load balancing or just getting all their particles to the MPI rank that owns their spatial locations?

Reply:
The latter. I will fix the wording.